### ISSUE TITLE
squared femoral epiphysis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7662]
**Preferred term label:**
squared femoral epiphysis / metacarpal

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
620022 
PMID: 24273071  3/3 (each)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@azankl
Is this a good term (also for metacarpals) or would this appearance be able to be described by an existing HPO term
https://pubmed.ncbi.nlm.nih.gov/24273071/, possibly Enlarged metacarpal epiphyses?

**Comment by azankl** [comment]
@pnrobinson I don't find the proximal femoral epiphyses in that article sufficiently 'squared' to justify that term. Would label this as 'Enlargement of the proximal femoral epiphysis HP:0003371'.
Have not seen or heard the term 'squared metacarpal epiphyses' before, but the metacarpal epiphyses in that paper indeed look squared, so maybe we should add this term, as a child of 'Abnormality of metacarpal epiphyses HP:0005913'

**Comment by pnrobinson** [comment]
@azankl returning this after a long time (sorry, last year we were relocating to Germany and I got way behind!)
It seems this is not really a well defined feature that would be present in several diseases, so I think let's not make an HPO term for now!
